src/java.base/windows/native/libjava/ProcessImpl_md.c

```diff
@@ -477,6 +498,15 @@ Java_java_lang_ProcessImpl_closeHandle(JNIEnv *env, jclass ignored, jlong handle
     return (jboolean) CloseHandle((HANDLE) handle);
 }
 
+/* SapMachine 2024-07-01: process group extension */
+JNIEXPORT void JNICALL
+Java_java_lang_ProcessImpl_terminateProcessGroup(JNIEnv *env, jclass ignored, jlong hJob)
+{
+    if (TerminateJobObject((HANDLE) hJob, 1) == 0){
```

Comment:
The space is fixed. I think the exit code 1, usually an error code, is fine because when a process is killed from outside I guess it can rather be regarded as non-normal exit.